src/Raven.Client/Documents/Commands/GetRevisionsBinEntryCommand.cs

```diff
@@ -12,13 +12,21 @@ public class GetRevisionsBinEntryCommand : RavenCommand<BlittableArrayResult>
     {
         private readonly long _etag;
         private readonly int? _pageSize;
+        private readonly string _continuationToken;
 
         public GetRevisionsBinEntryCommand(long etag, int? pageSize)
```

Comment:
Those parameters no longer match

Reply:
In case of non-sharded db I want the continuation token to be null, no?

Reply:
Changed